gradle.properties

```diff
@@ -10,5 +10,5 @@
 
 commonsCLIVersion=1.3.1
 commonsCSVVersion=1.5
-pravegaVersion=0.6.0
+pravegaVersion=0.7.0-50.ef40b45-SNAPSHOT
```

Comment:
this snapshot is slightly older to the current version of Pravega ( the last commit id c4e6dd8)